configs/sites/orion/compilers.yaml

```diff
@@ -40,20 +26,6 @@ compilers:
     - intel/2021.2
     environment: {}
     extra_rpaths: []
-- compiler:
```

Comment:
It would be good if we could also build and test with the oneAPI versions of the compilers. Did this section create problems?

Reply:
No, admittedly I've never tried the new compilers. I can put that back in.